executor/src/witgen/jit/prover_function_heuristics.rs

```diff
@@ -36,12 +38,23 @@ pub enum ProverFunctionComputation<'a> {
     /// The expression `f` in `query |i| std::prover::compute_from(Y, i, [X, ...], f)`,
     /// where f: (fe[] -> fe)
     ComputeFrom(&'a Expression),
+    /// Represents a call to `handle_query` (see `try_decode_handle_query`), where the
+    /// keys are the pc values (the first input column). The two two arguments to both input
```

Comment:
"two two arguments"

Reply:
I think this comment is missing some context here (not sure what "two arguments" or "input columns" are referring to)... maybe it should be elsewhere or have a bit more context?